src/services/commonSearchQueries.js

```diff
@@ -0,0 +1,76 @@
+import { Op } from 'sequelize';
+import {
+  tripRequests, status, tripTypes, users
+} from '../database/models';
+
+
+const query = (searchQueryParams) => ({
+  where: {
+    ...searchQueryParams.startDate && {
+      startDate: {
```

Comment:
@ivymwende I think we need to have a limitation in fetching data from DB. I found that any authenticated user can get all the data he/she may want that are not related to him/her.